fusor-ember-cli/app/styles/custom.scss

```diff
@@ -321,3 +321,10 @@ table.fusor-table {
   padding:0!important;
   cursor: pointer;
 }
+
+.eye-icon {
+  position: relative;
+  top: -20px;
+  float: right;
+  padding-right: 10px;
+}
```

Comment:
@rhamilto, this CSS seems to work. I only used relative positioning. I shrink my browser and it stays it the right position. Your previous comment was deleted when I overwrite this PR, but you mentioned that I should use absolute positioning for the parent.